internal/uvm/virtual_device.go

```diff
@@ -16,8 +16,14 @@ const (
 	VPCILocationPathIDType  = "vpci-location-path"
 	VPCIClassGUIDTypeLegacy = "class"
 	VPCIClassGUIDType       = "vpci-class-guid"
+	VPCIDeviceIDTypeLegacy  = "vpci"
+	VPCIDeviceIDType        = "vpci-instance-id"
 )
 
+// this is the well known channel type GUID for all assigned devices
+const vmbusChannelGUIDFormatted = "{44c4f61d-4444-4400-9d52-802e27ede19f}"
```

Comment:
Was this constant chosen by VMBus? If so, that would be good to mention here, as it's not clear if this is just something we have chosen or if it was determined elsewhere.

Reply:
Updated.